storagemarket/deal_execution.go

```diff
@@ -352,11 +361,17 @@ func (p *Provider) failDeal(pub event.Emitter, deal *types.ProviderDealState, er
 
 func (p *Provider) cleanupDeal(ctx context.Context, deal *types.ProviderDealState) {
 	_ = os.Remove(deal.InboundFilePath)
-	// ...
-	//cleanup resources here
+
+	// clean up tagged funds
 	err := p.fundManager.UntagFunds(ctx, deal.DealUuid)
 	if err != nil {
-		log.Errorw("untagging funds", "id", deal.DealUuid, "err", err)
+		log.Errorw("untagging funds", "uuid", deal.DealUuid, "err", err)
```

Comment:
I think we've used "id" everywhere for dealUUID - fine changing it to uuid, but if so let's do it everywhere (and in a separate PR)